src/components/AppNavigation/CalendarList/Trashbin.vue

```diff
@@ -131,10 +131,9 @@ export default {
 				color: calendar.color ?? uidToHexColor(calendar.displayname),
 			}))
 			const formattedCalendarObjects = this.objects.map(vobject => {
-				let eventSummary = t('calendar', 'Untitled event')
+				let eventSummary = t('calendar', 'Untitled item')
```

Comment:
```suggestion
				let eventSummary = t('calendar', 'Untitled element')
```

Reply:
Just to be in line with https://github.com/nextcloud/calendar/pull/3365#discussion_r676362126